plugins/org.eclipse.n4js/src/org/eclipse/n4js/resource/N4JSLinker.java

```diff
@@ -410,6 +411,10 @@ protected boolean hasLeafNodes(INode node) {
 	 */
 	@Override
 	protected EClass findInstantiableCompatible(EClass eType) {
+		// FIXME this is bad; #createProxy() above should be able to create proxies for abstract classes
```

Comment:
mind the fixme

Reply:
Good find! It is obsolete, will remove it.